dll/win32/msi/dialog.c

```diff
@@ -3094,7 +3094,7 @@ static UINT msi_dialog_directory_list( msi_dialog *dialog, MSIRECORD *rec )
     prop = MSI_RecordGetString( rec, 9 );
     control->property = msi_dialog_dup_property( dialog, prop, FALSE );
 
-    /* double click to activate an item in the list */
+    /* double-click to activate an item in the list */
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l117